src/commands/set_env_vars_for_maven.yml

```
@@ -0,0 +1,18 @@
+description:
```

```suggestion
description: >
```
Add the strip line break?